setup.py

```diff
@@ -36,7 +36,7 @@ def rtd_dependent_deps():
     "lint": lint_deps,
 }
 
-subprocess.Popen(protoc_cmd)
+output = subprocess.check_output(protoc_cmd)
```

Comment:
Nit: If we don't use the `output` variable, then let's just get rid of it. The `check_output` function will throw a `CalledProcessError` exception when the exit code is non-zero, which should be fine in this case